code-studio/scripts/build-commands.js

```diff
@@ -80,6 +81,9 @@ exports.bundle = function (config) {
     // Enables source map
     debug: true,
 
+    // Allow requiring jsx without specifying extension
+    extensions: ['.jsx'],
```

Comment:
When using the API, the argument is `extensions` (plural) [as documented here](https://github.com/substack/node-browserify#methods).